cmd/admin-info.go

```diff
@@ -175,6 +180,9 @@ func (u clusterStruct) String() (msg string) {
 	for _, srv := range u.Info.Servers {
 		// Check if MinIO server is not online ("Mode" field),
 		if srv.State != string(madmin.ItemOnline) {
+			if u.onlyOnline {
+				continue
+			}
```

Comment:
You should only skip printing. We want the side effects like `totalOfflineNodes++` and `OffDrives++`, etc.